server/webapp/WEB-INF/rails.new/app/assets/javascripts/lib/string-plus.js

```diff
@@ -15,12 +15,14 @@
  */
 
 define(['string', 'lodash'], function (s, _) {
+  var POSITIVE_INTEGER = /^\d+$/;
+
   var mixins = {
     defaultToIfBlank: function (value, defaultValue) {
       return s.isBlank(value) ? defaultValue : value;
     },
 
-    overrideToJSON: function (prop) {
+    collectionToJSON: function (prop) {
```

Comment:
Yes, I have it on my todo list. We probably need a generic "utils" object somewhere. I'll do it in a later commit, if that's OK?